SConstruct

```diff
@@ -238,13 +244,15 @@ def scons():
         exit(0)
 
     """Execute build"""
-    if os.path.exists('scons_local'):
+    if not os.path.exists('scons_local'):
+        need_scons_local()
+    else:
```

Comment:
Isn't this covered by the `ImportError` below?

Reply:
hmm, maybe it is.